.github/workflows/docs.yaml

```diff
@@ -22,38 +20,40 @@ on:
   schedule:
     - cron: '0 9 * * SUN'
 
+defaults:
+  run:
+    shell: bash -e {0}
+
+env:
+  PYTHON_VERSION: "3.11"
+
 jobs:
-  build_docs:
-    name: Documentation
+  pixi_lock:
+    name: Pixi lock
+    runs-on: ubuntu-latest
+    steps:
+      - uses: holoviz-dev/holoviz_tasks/pixi_lock@v0
+
+  docs_build:
+    name: Build Documentation
     runs-on: 'ubuntu-latest'
     timeout-minutes: 120
-    defaults:
-      run:
-        shell: bash -l {0}
     env:
-      MPLBACKEND: "Agg"
```

Comment:
This has been lost, add it to global env variables in pixi.toml

Reply:
OK